resources/views/components/transaction/page/token.blade.php

```diff
@@ -0,0 +1,41 @@
+@props(['transaction'])
+
+<x-general.page-section.container :title="trans('pages.transaction.tokens_transferred')">
+    <x-transaction.page.section-detail.row
+        :title="trans('pages.transaction.header.to')"
+        :transaction="$transaction"
+    >
+        <x-transaction.page.section-detail.address
+            :address="\App\Services\ContractPayload::decodeAddress($transaction->methodArguments()[0])"
+            class="inline-block"
+        />
+    </x-transaction.page.section-detail.row>
+
+    <x-transaction.page.section-detail.row
+        :title="trans('pages.transaction.header.amount')"
+        :value="ExplorerNumberFormatter::weiToArk(\App\Services\ContractPayload::decodeUnsignedInt($transaction->methodArguments()[1]))"
+    >
+        <x-transaction.amount :transaction="$transaction" />
+    </x-transaction.page.section-detail.row>
+
+    @if (Network::canBeExchanged())
+        @if (ExplorerNumberFormatter::isFiat(Settings::currency()) && ExchangeRate::convertNumerical($transaction->amountWithFee(), $transaction->model()->timestamp) < 0.01)
+            <x-transaction.page.section-detail.row
+                :title="trans('pages.transaction.header.value')"
+                :value="'<'.ExplorerNumberFormatter::currency(0.01, Settings::currency())"
```

Comment:
this should be hidden if the token itself has no monetary value. This will be irregardless of whether the network itself has monetary value 🤔

Reply:
I'll remove it completely for now and handle it later on. I'm not too sure how to determine if it has monetary value just yet :thinking: since we currently use jobs to cache the monetary value of ARK

Reply:
https://app.clickup.com/t/86dvbg010